OneCommunityGlobal/HighestGoodNetworkApp
# Nathan - removed <p> and linted

# Description
Removes validateDOMNesting warning: \<div\> cannot appear as a descendant of \<p\>.

## Main changes explained:
- Removed a \<p\> wrapper
- Linting changes on the rest of the file

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. log as admin user
4. go to dashboard
5. verify that warning does not appear in the dev tools console

## Screenshots or videos of changes:
Before:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/1f859840-32b9-4c5f-afcc-c2ba0843b8c9)
After:
![image](https://github.com/OneCommunityGlobal/HighestGoodNetworkApp/assets/23086054/c707cb2e-7c73-4da8-93d5-7b8c39898c4f)
